src/basic/parse-util.c

```diff
@@ -779,3 +780,12 @@ int parse_loadavg_fixed_point(const char *s, loadavg_t *ret) {
 
         return store_loadavg_fixed_point(i, f, ret);
 }
+
+int parse_slot(const char *s, int *ret, int nonzero) {
```

Comment:
this is pretty specific to the ifnaming thing, and only has a user there. please keep this in netif-naming-scheme.c

also, please use C99 `bool` booleans please

Reply:
Shouldn't that go to udev-builtin-net_id.c ?

Reply:
of course, sorry